Lib/test/test_random.py

```diff
@@ -747,6 +763,15 @@ def test_choices_algorithms(self):
         c = self.gen.choices(population, cum_weights=cum_weights, k=10000)
         self.assertEqual(a, c)
 
+    def test_randbytes(self):
+        super().test_randbytes()
+
+        # Mersenne Twister randbytes() is deterministic
+        # and does not depend on the endian
+        self.gen.seed(8675309)
+        self.assertEqual(self.gen.randbytes(16),
```

Comment:
Call `randbytes()` yet one time. We should be sure that it does not consume more bits from the Mersenne Twister generator than necessary. It would be nice to test it also with the number of bytes not divisible by 4, to test what bits from non-complete word are used (highest or lowest).

Reply:
I added more tests on randbytes(2) and randbytes(3).